CHANGELOG.md

```diff
@@ -6,6 +6,8 @@
     Click to see more.
   </summary>
 
```

Comment:
you need to move the closing `</details>` tag from down below above this so that we reset the unreleased section of the change log @dalvarez

Reply:
oh whoops - good catch! fixed